client/gameObjects/Skeleton.js

```diff
@@ -1,5 +1,5 @@
 import Phaser from "phaser";
-import HealthBar from "./HealthBar";
+import { HealthBar, EnergyBar } from "./Bar";
```

Comment:
This could be sorted alphabetically

Reply:
This could be done automatically, I have added corresponding ESLint rule.